### PR TITLE
OCPBUGS-42102#update to Standard Next

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions:
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-42102

Link to docs preview:

- https://83375--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/installing-ibm-cloud-account.html#installation-cis-ibm-cloud_installing-ibm-cloud-account
- https://83375--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/installing-ibm-cloud-account.html#installation-dns-ibm-cloud_installing-ibm-cloud-account

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-cis-ibm-cloud.adoc
Comment (gcs278) [inline modules/installation-cis-ibm-cloud.adoc:1]: I see one more place we should change "standard" to "standard next": https://83375--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/installing-ibm-cloud-account.html#installation-cis-ibm-cloud_installing-ibm-cloud-account:~:text=At%20a%20minimum%2C%20a%20Standard%20plan%20is%20required%20for%20DNS%20Services%20to%20manage%20the%20cluster%20subdomain%20and%20its%20DNS%20records.

```
At a minimum, a Standard plan is required for DNS Services to manage the cluster subdomain and its DNS records.
```

I think it's in a different module.

**Comment by brendan-daly-red-hat** [comment]
@MayXuQQ, Please review the draft update and confirm if this change applies to 4.14+. Thanks.

**Comment by gcs278** [comment]
Thanks for the bug fix!
/lgtm

It sounds like this probably should be backported if I read the IBM [update](https://docs.openshift.com/container-platform/4.16/installing/installing_ibm_cloud_public/installing-ibm-cloud-account.html#installation-cis-ibm-cloud_installing-ibm-cloud-account) correctly.
```
Starting 30 April 2023, the CIS current Standard plan will no longer be available for new instances.
```
Regardless of our OpenShift version, sounds like "Standard" is now "Standard Next". I think this change can be backported as-is to all supported OpenShift versions.

**Comment by MayXuQQ** [comment]
based on my test
1. instance-create under **Defaut** group. 
`ibmcloud target -g Default`

2. https://cloud.ibm.com/docs/cis?topic=cis-getting-started#configure-your-name-servers-with-the-registrar-or-existing-dns-provider 

>  After you configure your registrar or DNS provider, it can take up to **24** hours for the changes to take effect.

**Comment by brendan-daly-red-hat** [comment]
> based on my test
> 
> 1. instance-create under **Defaut** group.
>    `ibmcloud target -g Default`
> 2. https://cloud.ibm.com/docs/cis?topic=cis-getting-started#configure-your-name-servers-with-the-registrar-or-existing-dns-provider
> 
> > After you configure your registrar or DNS provider, it can take up to **24** hours for the changes to take effect.

For the modules [Using IBM Cloud Internet Services for DNS resolution](https://83375--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/installing-ibm-cloud-account.html#installation-cis-ibm-cloud_installing-ibm-cloud-account) and [Using IBM Cloud DNS Services for DNS resolution](https://83375--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/installing-ibm-cloud-account.html#installation-dns-ibm-cloud_installing-ibm-cloud-account), after the "At a minimum, a Standard Next..." text add a note?
"Note: After you configure your registrar or DNS provider, it can take up to **24** hours for the changes to take effect."

**Comment by MayXuQQ** [comment]
I agree to add a note.
> "Note: After you configure your registrar or DNS provider, it can take up to 24 hours for the changes to take effect."

**Comment by brendan-daly-red-hat** [comment]
@MayXuQQ, Thanks. PTAL the updated preview links.

File: modules/installation-cis-ibm-cloud.adoc [hunk 1]
```diff
@@ -49,9 +49,14 @@ endif::ibm-power-vs[]
 +
 [source,terminal]
 ----
-$ ibmcloud cis instance-create <instance_name> standard <1>
+$ ibmcloud cis instance-create <instance_name> standard-next <1>
 ----
-<1> At a minimum, a `Standard` plan is required for CIS to manage the cluster subdomain and its DNS records.
+<1> At a minimum, a `Standard Next` plan is required for CIS to manage the cluster subdomain and its DNS records.
+
```
Comment (agantony) [inline modules/installation-cis-ibm-cloud.adoc:55]:
```suggestion
+
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

> The numbering in the preview is wrong; you need to use a "+" before the "note" to correct it. I have added a suggestion.

**Pull request merged**